great-idea/index.html

```diff
@@ -14,10 +15,94 @@
   <![endif]-->
 </head>
 
-  <body>
+<body>
+  <nav>
+    <a href="services.html">Services</a>
+    <a href="#">Product</a>
+    <a href="#">Vision</a>
+    <a href="#">Features</a>
+    <a href="#">About</a>
+    <a href="#">Contact</a>
+    <img class="logo" src="img/logo.png" alt="Great Idea! Company logo." />
+  </nav>
+
+  <header>
+    <h1>Innovation<br />On<br />Demand</h1>
+    <img src="img/header-img.png" alt="Image of a code snippet." />
+  </header>
+  <button>Get Started</button><br /><br />
```

Comment:
Try and think about where you need to add additional divs in-order to help with styling. 

For example A sensible way to structure the landing page could be something like this:

```
<header>
    <div class="text-section">
        <h1>Innovation<br /> On<br />Demand</h1>
        <button>Get started</button>
    </div>
    <div class="image-section">
        <img src="...." />
    </div>
</header>
```

This then allows you to use flex-box to easily position your elements within these divs.

Let me know if this doesn't make sense!

Try using the pesticide app we talked about to have a look at some website you like. Have a look and see how they've laid out their boxes to achieve their layouts.